packages/botonic-core/src/hubtype-service.js

```diff
@@ -149,7 +146,6 @@ export class HubtypeService {
       await this.init(user)
     } catch (e) {
       this.handleUnsentInput(message)
```

Comment:
* we discussed this in the past. any reason for postMessage function having 2 try/catch?
* Remove return Promise.resolve() and //eslint-disable-next-line consistent-return
* convert axios .then.catch to await to simplify code
* You will then only need 1 this.handleUnsentInput calls instead of 3

Reply:
solved, I think you will like this refactor :)